analyses/snv-callers/lancet-wxs-tests/explore-tcga-pbta.Rmd

```diff
@@ -0,0 +1,591 @@
+
+```{r}
+lancet %>%
+  ggplot2::ggplot(ggplot2::aes(x = log10(t_depth), color = dataset)) +
+  ggplot2::geom_density()
+```
+
+```{r}
+mutect %>%
+  ggplot2::ggplot(ggplot2::aes(x = log10(t_depth), color = dataset)) +
+  ggplot2::geom_density()
+```
+
+```{r}
+strelka %>%
+  ggplot2::ggplot(ggplot2::aes(x = log10(t_depth), color = dataset)) +
+  ggplot2::geom_density()
+```
+
+## How does the TMB comparison look for each caller by itself? 
+
+Lancet only TMB:
+
+```{r}
+lancet_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_lancet,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_lancet,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"), # There's no TCGA WGS samples, this is just a place holder and won't be used.
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+tmb_cdf_plot(lancet_tmb, plot_title = "Lancet")
+```
+
+[Lancet TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Lancet.png)
+
+Mutect only TMB:
+
+```{r}
+mutect_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_mutect,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_mutect,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(mutect_tmb, plot_title = "Mutect2")
+```
+
+[Mutect2 TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Mutect2.png)
+
+Strelka only TMB:
+
+```{r}
+strelka_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_strelka,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_strelka,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(strelka_tmb, plot_title = "Strelka2")
+```
+
+[Strelka2 TMB](plots/tcga-vs-pbta-plots/tmb-cdf-Strelka2.png)
+
+Consensus TMB:
+
+```{r}
+consensus_tmb <- dplyr::bind_rows(list(
+  pbta = calculate_tmb(pbta_consensus,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(data_dir, "intersect_cds_lancet_WXS.bed")
+  ),
+  tcga = calculate_tmb(tcga_consensus,
+    bed_wgs = file.path(data_dir, "intersect_cds_lancet_strelka_mutect_WGS.bed"),
+    bed_wxs = file.path(ref_dir, "intersect_cds_gencode_liftover_WXS.bed")
+  )
+), .id = "datasets")
+
+# Make a CDF plot
+tmb_cdf_plot(consensus_tmb, plot_title = "Consensus")
```

Comment:
This plot doesn't look so bad! Is this different from the main tcga-pbta comparison somehow?

Reply:
Yes. The BED files here were wrong #566 was not implemented yet and #568 is still at large. So the PBTA TMBs got larger because the BED files got smaller.